vars/createOpenshiftDockerSecret.groovy

```diff
@@ -0,0 +1,22 @@
+def call(String secretName='dockerhub', String dockerServer, String username, String password, String email) {
```

Comment:
It seems to not matter, and it can figure it out. I agree that having it as the last param seems more normal, so I'll change to that